oauth2_http/javatests/com/google/auth/oauth2/ServiceAccountCredentialsTest.java

```diff
@@ -338,7 +339,12 @@ public void fromJSON_getProjectId() throws IOException {
     transportFactory.transport.addServiceAccount(SA_CLIENT_EMAIL, ACCESS_TOKEN);
     GenericJson json =
         writeServiceAccountJson(
-            SA_CLIENT_ID, SA_CLIENT_EMAIL, SA_PRIVATE_KEY_PKCS8, SA_PRIVATE_KEY_ID, PROJECT_ID);
+            SA_CLIENT_ID,
```

Comment:
SA == SERVICE_ACCOUNT?
avoid abbreviations

Reply:
I would assume so, I did not add this, but I can expand.